metadata.json

```diff
@@ -42,6 +42,14 @@
       ]
     },
     {
+      "operatingsystem": "OpenBSD",
+      "operatingsystemrelease": [
+        "5.4",
```

Comment:
Would you mind posting a gist of the output of facter on your openbsd system? It helps with the metadata and with supporting openbsd with other projects.